### ISSUE TITLE
Trouble with `opam submit` and v-prefixed tag

### ISSUE THREAD

**Issue by hcarty** [samoht/dune-release#39]
Using dune-release 0.2.0 I was able to get https://github.com/hcarty/ocaml-gdal/releases/tag/v0.10.0 without any trouble by going through the release steps up through `dune-release opam pkg`.  Yay and thank you!

Unfortunately `dune-release opam submit` fails with this error:
```
dune-release: [ERROR] /home/hcarty/projects/ocaml-gdal/_build/gdal.v0.10.0
                      does not exist, did you run:
                        dune-release opam pkg -n gdal
```
`/home/hcarty/projects/ocaml-gdal/_build/gdal.v0.10.0` does not exist but `/home/hcarty/projects/ocaml-gdal/_build/gdal.0.10.0` does.  The `v` was trimmed from the version number during `dune-release opam pkg` but not during `dune-release opam submit`.

**Comment by samoht** [comment]
Can you try with master? @let-def submitted #41 recently which should probably help. I

**Comment by hcarty** [comment]
Definitely! I should be able to test it a little later today.

**Comment by hcarty** [comment]
Small bug while testing:
```
Upgrading configuration files: /home/hcarty/.dune => /home/hcarty/.config/dune
mv: cannot move '/home/hcarty/.dune' to '/home/hcarty/.config/dune': No such file or directory
dune-release: [ERROR] run ['mv' '/home/hcarty/.dune'
                      '/home/hcarty/.config/dune']: exited with 1
```
The error happened because this VM didn't have a `~/.config` directory.

**Comment by hcarty** [comment]
@samoht This issue seems to be fixed!  Thank you @let-def for #41.

Unfortunately `dune-release` prompted me for my github credentials rather than using the token I'd setup previously.  I don't think that will work due to 2fa.

**Comment by hcarty** [comment]
I was able to get through the submission by editing `~/.config/dune/release.yml` to use `git@github.com:...` rather than `https://github.com/...` for the `remote:` field.

Then I got one more innocuous failure due to running in a headless VM:
```
dune-release: [ERROR] run ['xdg-open'
                      'https://github.com/ocaml/opam-repository/pull/12315']:
                      exited with 127
```
If the browser open fails, it may be worth printing out the PR's location with a friendly "I can't open a browser window for you, but here's your shiny new PR's location if you want to check it out: ..."

**Comment by samoht** [comment]
Should be fixed by #48

**Comment by hcarty** [comment]
Thank you very much!
